src/tunnel.cc

```diff
@@ -477,9 +480,6 @@ TunnelStateData::retryOrBail(const char *context)
 
     /* bail */
 
-    if (request)
-        request->hier.stopPeerClock(false);
-
```

Comment:
Is there a reason to doubt request existence here? The constructor asserts its existence and our startPeerClock() uses that assumption. If we can make this and other stopPeerClock() calls in TunnelStateData unconditional as well, please do that in this PR.

Reply:
This change request is not addressed by the current PR code. Not sure why I resolved it earlier.

Reply:
Done.